ckanext/harvest/logic/action/update.py

```diff
@@ -482,6 +484,9 @@ def harvest_jobs_run(context, data_dict):
                     # status
                     get_action('harvest_source_reindex')(
                         context, {'id': job_obj.source.id})
+
+                    if toolkit.asbool(config.get('ckan.harvest.status_mail.errored')):
```

Comment:
As discussed, we should only set the email if the job errored.

Reply:
I think it would be cleaner to move 569-581 into a separate function (to check it a particular job errored) and include the output of that here, if `ckan.harvest.status_mail.errored` is True.

Other than that, I think it looks good.

@stefina I realise that this is an old PR, but it's really useful functionality .. do you think you'd be up for the small change suggested?

Reply:
@rossjones Good point! I refactored it accordingly.